manticore/wasm/cli.py

```diff
@@ -18,9 +18,12 @@ def wasm_main(args, _logger):
         policy=args.policy,
     )
 
-    if consts.profile:
-        profiler = Profiler()
-        m.register_plugin(profiler)
+    try:
```

Comment:
Is this causing an AttributeError to be thrown? The profile flag is created in ethereum/cli.py, which is definitely weird, but it's never caused problems before. Might we instead copy the following line into wasm/cli.py as well?

https://github.com/trailofbits/manticore/blob/2152023f390a7dc45cf16cd5cc283d0cfa93b8ee/manticore/ethereum/cli.py#L30

Reply:
Hmmm. Yeah, I think I just forgot to remove this change before committing. I ran into something where this errored on the missing attribute, but I can't seem to reproduce it now 🤷 

I think adding that line to `wasm/cli.py` would be good in any case though 👍  (Had to delete a corresponding line in the test file)

Reply:
The process for adding config options seems to be a bit delicate, so I've added some logic to allow the same exact config option to be "added" twice... I'll break this out into a separate PR

Reply:
Opened in #2397 and removed those changes in this PR